bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -327,22 +331,27 @@ public void channelLinked(ChannelUID channelUID) {
 
     @Override
     public void onSensorRemoved(@Nullable HueBridge bridge, FullSensor sensor) {
-        if (sensor.getId().equals(sensorId)) {
+        if (sensor.getId().equals(sensorId)) { // TODO Maybe this check is obsolete
```

Comment:
With your changes, this check becomes useless (it was useful before).

Reply:
Removed